react/components/Input/index.js

```diff
@@ -264,7 +265,13 @@ class Input extends Component {
           )}
         </div>
         {errorMessage && (
-          <div className="c-danger t-small mt3 lh-title">{errorMessage}</div>
+          <div className="c-danger flex items-center t-small mt3 lh-title">
+            {/* use flex to align svg vertically */}
+            <span className="mr2 flex">
```

Comment:
Can you add a vtex-input__error-icon class here so people can customize it?
```suggestion
            <span className="vtex-input__error-icon mr2 flex">
```

Reply:
Ok!